evm/src/main/java/org/hyperledger/besu/evm/processor/AbstractMessageProcessor.java

```diff
@@ -192,6 +208,9 @@ public void process(final MessageFrame frame, final OperationTracer operationTra
       codeExecute(frame, operationTracer);
 
       if (frame.getState() == MessageFrame.State.CODE_SUSPENDED) {
+        if (operationTracer != null) {
```

Comment:
Suspended does not mean the context is exiting.  Really what it means is that we are starting a new sub-context and the particulars of how this state machine is written we need to restart from the beginning.

This should be removed.